### PR TITLE
Improve DB query

### PR DESCRIPTION
**Pull request by dyktus**

### CODE REVIEW

File: src/cli/CleanFaresCommand.ts [hunk 1]
```diff
@@ -44,6 +44,8 @@ export class CleanFaresCommand implements CLICommand {
 
   private readonly indexes = [
     "CREATE INDEX fare_flow_id ON fare (flow_id)",
+    "ALTER TABLE toc_specific_ticket ADD COLUMN IF NOT EXISTS hash VARCHAR(255); UPDATE toc_specific_ticket SET hash=COALESCE(ticket_code, restriction_code, restriction_flag, direction, end_date);",
+    "CREATE INDEX toc_specific_ticket_toc_key ON toc_specific_ticket (hash)"
```
Comment (matiangul) [inline src/cli/CleanFaresCommand.ts:48]: You probably need some procedure that will update this hash on INSERT and UPDATE

Comment (matiangul) [reply src/cli/CleanFaresCommand.ts:48]: Take a look at this construct:

https://gitlab.com/assertis/order-service/-/blob/master/migrations/20190523093505_indexes_for_leg_loading.php#L19

Comment (dyktus) [reply src/cli/CleanFaresCommand.ts:48]: No need to do it. 

1) We do the update of fares all Inserts, updates and removal
2) We run CleanFaresCommand (so after data update we'll always update toc_specific_ticket)

Comment (matiangul) [reply src/cli/CleanFaresCommand.ts:48]: Ok then go for it ;)

**Pull request merged**